app/Models/Dictionary.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Dictionary extends Model
+{
+    use HasFactory;
+
+    public function terms()
+    {
+        return $this->hasMany(DictionaryTerm::class);
+    }
+
+    public static function isRole($role): bool
+    {
+        return $role->key == "role";
```

Comment:
w stringach lepiej używać ' a nie ". Stringi powinny być przeniesione do cosntów

Reply:
W sumie z tego co czytałem, to rzadko się zdarza, żeby "" było mniej wydajne od '', także to nie problem, że są w "", ale możemy przyjąć taką konwencję, że stringi w ''.

Reply:
dokładnie przyjmijmy pls stringi w ' a jak jest jakaś duża potrzeba czy cuś